README.md

```diff
@@ -1,30 +1,87 @@
-# Aliucord
-Discord Android app mod inspired by desktop Discord client mods. Unlike other mods, you don't need to rebuild the apk on PC, because Aliucord is hooking at runtime using the [Pine](https://github.com/canyie/pine) hook framework.
+<h1 align="center">Aliucord</h1>
+<p align= "center">
+  <a href="https://discord.gg/EsNDvBaHVU">
+    <img alt="Discord" src="https://img.shields.io/discord/811255666990907402?color=%2300C853&label=Support%20Server&logo=discord&logoColor=%2300C853&style=for-the-badge">
+  </a>
+</p>
 
-⚠ **Info**: Aliucord doesn't support `x86` and `x86_64` architectures, because Pine supports `arm` and `arm64` only.
+## What is Aliucord?
 
-**Supported Android versions**: 7.0 or higher (sdk 24)
+A modification for the Android Discord app inspired by desktop client modifications. 
+Unlike other Android Discord app modifications, you don't need to rebuild the APK when adding or removing plugins, because Aliucord hooks at runtime using the [Pine](https://github.com/canyie/pine) framework.
 
-**Supported Discord version**: 80.2 - Alpha (80202)
+## Important Information
 
-[Support Server](https://discord.gg/EsNDvBaHVU)
+⚠ **Please Read**
 
-### Download
+Aliucord does not support the `x86` or `x86_64` architectures, because Pine only supports `arm` and `arm64`.
+
+**Supported Android version(s)** 
+
+- 7.0+ (SDK 24+)
+
+**Supported Discord version(s)**
+
+- 80.2 - Alpha (80202)
+
+## Build/Installation Instructions
+
+### Installation
 1. Download [Installer-release.apk](https://github.com/Aliucord/Aliucord/raw/builds/Installer-release.apk) from the `builds` branch
-2. Install it
-3. Open it, click Install, choose the Download option (recommended)
-4. Wait for it to finish its job
-5. Click Install once prompted to install Aliucord
+2. Find it via your preferred files viewing app, then install it
+3. Open the newly installed "Aliucord Installer" app from your app drawer
+4. Click "Install", then choose the "Download" option 
+
+- *psst... only choose the other options if you know what you're doing!*
```

Comment:
```suggestion
  - *psst... only choose the other options if you know what you're doing!*
```

indent this so it shows as sub option of 4 and doesn't interrupt the list

Reply:
I assume 7fabf5e fixes this?